all-products.html

```diff
@@ -11,7 +11,7 @@
 			<nav name="top">
 				<ul>
 					<li><a href="quiz.html">skin type quiz</a></li>
-					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+					<li class="dropdown"><a href="all-products.html" class="current">products &#9662;</a>

```

Comment:
Updated the active underline in the navbar when the user is on any product related page.